src/main/java/uk/gov/pay/webhooks/message/dao/WebhookMessageDao.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.pay.webhooks.message.dao;
+
+import io.dropwizard.hibernate.AbstractDAO;
+import org.hibernate.SessionFactory;
+import uk.gov.pay.webhooks.message.dao.entity.WebhookMessageEntity;
+
+import javax.inject.Inject;
+
+public class WebhookMessageDao extends AbstractDAO<WebhookMessageEntity> {
+
+    @Inject
+    public WebhookMessageDao(SessionFactory factory) {
+        super(factory);
+        }
```

Comment:
Can we add a line break between the methods here?

Reply:
👍